query-planner-js/src/config.ts

```diff
@@ -83,7 +83,19 @@ export type QueryPlannerConfig = {
      * query plans).
      */
     maxEvaluatedPlans?: number,
+
+      /**
+   * Before creating query plans, we need to evaluate all the possible paths to access every field:
+   * through which subgraph providing it, through which type in union or type implementing an
+   * interface. The number of paths generated can grow if the schema or query are complex,
+   * and that will increase the time spent planning.
+   *
+   * The default value is set to 10000
+   */
```

Comment:
Adjusted the indentation and phrasing a bit.
```suggestion
    /**
     * Before creating query plans, for each path of fields in the query we compute all the
     * possible options to traverse that path via the subgraphs. Multiple options can arise because
     * fields in the path can be provided by multiple subgraphs, and abstract types (i.e. unions
     * and interfaces) returned by fields sometimes require the query planner to traverse through
     * each constituent object type. The number of options generated in this computation can grow
     * large if the schema or query are sufficiently complex, and that will increase the time spent
     * planning.
     * 
     * This config allows specifying a per-path limit to the number of options considered. If any
     * path's options exceeds this limit, query planning will abort and the operation will fail.
     *
     * The default value is null, which specifies no limit.
     */
```